feat-content.html

```diff
@@ -31,5 +23,13 @@ <h4 class="feat-content-title">Introduction to Dockstore Webinar</h4>
             <p class="feat-card-description">A simple overview of the Dockstore platform and features. Covers the fundamentals of finding workflows and provides examples of running analysis with sample data.</p>
         </a>
     </div>
+        
+    <div class="feat-content-card">
+        <a class="cardClick" target="_blank" rel="noopener noreferrer"
+           href="https://www.youtube.com/watch?v=VE2J6-4wSNM">
+            <h4 class="feat-content-title">Data Analysis on AnVIL: Use Cases</h4>
+            <p class="feat-card-description">A brief introduction on Data Analyis on AnVIL highlighting some example use cases and with short introduction of tools and tutorials.</p>
```

Comment:
I think the video mainly highlights the use cases/workflows rather than actually giving an intro to the data analysis or a tutorial. So should probably clarify the description a bit. 
'A brief overview of data analysis use cases on the AnVIL platform, highlighting workflows available on Dockstore' 

might need to check my grammar though

Reply:
Thanks for reviewing. I updated with your suggestions and ran the result by Beth. Ended up with a new desc.